### ISSUE TITLE
Can't override Server header

### ISSUE THREAD

**Issue by CKarper** [trezm/Thruster#106]
I tried to set a `Server` header, but it was getting doubled with the [static header text](https://github.com/trezm/Thruster/blob/master/thruster-core/src/response.rs#L60) in the `Response`.   Having `Thruster` as the default is a great idea, but if it could go through the regular `set` method on the response, that would allow a program to `remove` it when desired.

Standard approach in my microservices is (or at least what I expected to work):
```rust
#[middleware_fn]
async fn server(mut context: Ctx, next: MiddlewareNext<Ctx>) -> Ctx {
	context = await!(next(context));
	context.set("Server", &format!("{} v{}", PKG_NAME, PKG_VERSION));
	context
}
```

**Comment by trezm** [comment]
Good idea, I'll start working on an update for this.

**Comment by CKarper** [comment]
Thank you!

**Comment by trezm** [comment]
Fixed in #107
